### ISSUE TITLE
Update Lexend with Italics

### ISSUE THREAD

**Issue by RosaWagner** [google/fonts#4237]
**Font Project Git Repo URL:**
https://github.com/googlefonts/lexend

**Super short description of the upgrade:**
Add Italic

cc @otaviopardo

**Comment by ithinkihaveacat** [comment]
Related issue: https://github.com/googlefonts/lexend/issues/14.

**Comment by emmamarichal** [comment]
@RosaWagner you can find the fonts exported here: https://github.com/emmamarichal/lexend

remarks: 
- I didn't manage to solve this one: 🔥 FAIL: Validate STAT particle names and values match the fallback names in GFAxisRegistry.
- Also, the font was not exporting because of a problem with the glyphs `estimate` (compatibility problem). The glyphs looked normal, I couldn't figure out the problem. For the moment I just put it in "not exported".
- For now I set up the fonts with a slant axis because that's how it was delivered. But since the axis only goes in one direction (0 to -15), it might be better to set an italic.

## Fontbakery report

Fontbakery version: 0.8.10

<details><summary><b>[1] Family checks</b></summary><div><details><summary>ℹ <b>INFO:</b> Check axis ordering on the STAT table.  (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/STAT/axis_order">com.google.fonts/check/STAT/axis_order</a>)</summary><div>


* ℹ **INFO** From a total of 1 font files, 0 of them (0.00%) lack a STAT table.

	And these are the most common STAT axis orderings:
	('HEXP-slnt-wght', 1) [code: summary]
</div></details><br></div></details><details><summary><b>[15] Lexend[HEXP,slnt,wght].ttf</b></summary><div><details><summary>🔥 <b>FAIL:</b> Check variable font instances (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fvar_instances">com.google.fonts/check/fvar_instances</a>)</summary><div>


* 🔥 **FAIL** fvar instances are incorrect:
- Add missing instances

| Name | current | expected |
| :--- | :--- | :--- |
| Thin Italic | N/A | wght=100.0, HEXP=0.0, slnt=-15.0 |
| Thin | wght=100.0, HEXP=0.0, slnt=0.0 | wght=100.0, HEXP=0.0, slnt=0.0 |
| ExtraLight Italic | N/A | wght=200.0, HEXP=0.0, slnt=-15.0 |
| ExtraLight | wght=200.0, HEXP=0.0, slnt=0.0 | wght=200.0, HEXP=0.0, slnt=0.0 |
| Light Italic | N/A | wght=300.0, HEXP=0.0, slnt=-15.0 |
| Light | wght=300.0, HEXP=0.0, slnt=0.0 | wght=300.0, HEXP=0.0, slnt=0.0 |
| Italic | N/A | wght=400.0, HEXP=0.0, slnt=-15.0 |
| Regular | wght=400.0, HEXP=0.0, slnt=0.0 | wght=400.0, HEXP=0.0, slnt=0.0 |
| Medium Italic | N/A | wght=500.0, HEXP=0.0, slnt=-15.0 |
| Medium | wght=500.0, HEXP=0.0, slnt=0.0 | wght=500.0, HEXP=0.0, slnt=0.0 |
| SemiBold Italic | N/A | wght=600.0, HEXP=0.0, slnt=-15.0 |
| SemiBold | wght=600.0, HEXP=0.0, slnt=0.0 | wght=600.0, HEXP=0.0, slnt=0.0 |
| Bold Italic | N/A | wght=700.0, HEXP=0.0, slnt=-15.0 |
| Bold | wght=700.0, HEXP=0.0, slnt=0.0 | wght=700.0, HEXP=0.0, slnt=0.0 |
| ExtraBold Italic | N/A | wght=800.0, HEXP=0.0, slnt=-15.0 |
| ExtraBold | wght=800.0, HEXP=0.0, slnt=0.0 | wght=800.0, HEXP=0.0, slnt=0.0 |
| Black Italic | N/A | wght=900.0, HEXP=0.0, slnt=-15.0 |
| Black | wght=900.0, HEXP=0.0, slnt=0.0 | wght=900.0, HEXP=0.0, slnt=0.0 | [code: bad-fvar-instances]
</div></details><details><summary>🔥 <b>FAIL:</b> Validate STAT particle names and values match the fallback names in GFAxisRegistry.  (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/STAT/gf_axisregistry">com.google.fonts/check/STAT/gf_axisregistry</a>)</summary><div>


* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Deca' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Exa' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Giga' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Mega' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Peta' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Tera' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
* 🔥 **FAIL** On the font variation axis 'HEXP', the name 'Zetta' is not among the expected ones (Default) according to the Google Fonts Axis Registry. [code: invalid-name]
</div></details><details><summary>⚠ <b>WARN:</b> Checking OS/2 achVendID. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vendor_id">com.google.fonts/check/vendor_id</a>)</summary><div>


* ⚠ **WARN** OS/2 VendorID value 'NONE' is not yet recognized. If you registered it recently, then it's safe to ignore this warning message. Otherwise, you should set it to your own unique 4 character code, and register it with Microsoft at https://www.microsoft.com/typography/links/vendorlist.aspx
 [code: unknown]
</div></details><details><summary>⚠ <b>WARN:</b> Check Google Fonts glyph coverage. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage">com.google.fonts/check/glyph_coverage</a>)</summary><div>


* ⚠ **WARN** GF_TransLatin_Arabic is almost fulfilled. Missing codepoints:

	- 0x1E34 (LATIN CAPITAL LETTER K WITH LINE BELOW)


	- 0x1E35 (LATIN SMALL LETTER K WITH LINE BELOW)


	- 0x1E96 (LATIN SMALL LETTER H WITH LINE BELOW)
 

	- And 0x02BD (MODIFIER LETTER REVERSED COMMA)
 [code: missing-codepoints]
</div></details><details><summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences">com.google.fonts/check/kerning_for_non_ligated_sequences</a>)</summary><div>


* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:

	- f + f

	- f + i 

	- And i + l [code: lacks-kern-info]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure fonts have ScriptLangTags declared on the 'meta' table. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/meta/script_lang_tags">com.google.fonts/check/meta/script_lang_tags</a>)</summary><div>


* ⚠ **WARN** This font file does not have a 'meta' table. [code: lacks-meta-table]
</div></details><details><summary>⚠ <b>WARN:</b> Check font contains no unreachable glyphs (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/unreachable_glyphs">com.google.fonts/check/unreachable_glyphs</a>)</summary><div>


* ⚠ **WARN** The following glyphs could not be reached by codepoint or substitution rules:

	- gravecombinator

	- peseta.bold

	- sterling.base

	- uni004A0301

	- uni006A0301

	- uni20A6.bold

	- uni20A9.bold 

	- And uni20B1.bold
 [code: unreachable-glyphs]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure dotted circle glyph is present and can attach marks. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/dotted_circle">com.google.fonts/check/dotted_circle</a>)</summary><div>


* ⚠ **WARN** No dotted circle glyph present [code: missing-dotted-circle]
</div></details><details><summary>ℹ <b>INFO:</b> Show hinting filesize impact. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/hinting_impact">com.google.fonts/check/hinting_impact</a>)</summary><div>


* ℹ **INFO** Hinting filesize impact:

 |               | Lexend[HEXP,slnt,wght].ttf          |
 |:------------- | ---------------:|
 | Dehinted Size | 551.7kb |
 | Hinted Size   | 551.7kb   |
 | Increase      | 24 bytes      |
 | Change        | 0.0 %  |
 [code: size-impact]
</div></details><details><summary>ℹ <b>INFO:</b> Font has old ttfautohint applied? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/old_ttfautohint">com.google.fonts/check/old_ttfautohint</a>)</summary><div>


* ℹ **INFO** Could not detect which version of ttfautohint was used in this font. It is typically specified as a comment in the font version entries of the 'name' table. Such font version strings are currently: ['Version 1.010;gftools[0.9.23]'] [code: version-not-detected]
</div></details><details><summary>ℹ <b>INFO:</b> EPAR table present in font? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/epar">com.google.fonts/check/epar</a>)</summary><div>


* ℹ **INFO** EPAR table not present in font. To learn more see https://github.com/googlefonts/fontbakery/issues/818 [code: lacks-EPAR]
</div></details><details><summary>ℹ <b>INFO:</b> Is the Grid-fitting and Scan-conversion Procedure ('gasp') table set to optimize rendering? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/gasp">com.google.fonts/check/gasp</a>)</summary><div>


* ℹ **INFO** These are the ppm ranges declared on the gasp table:

PPM <= 65535:
	flag = 0x0F
	- Use grid-fitting
	- Use grayscale rendering
	- Use gridfitting with ClearType symmetric smoothing
	- Use smoothing along multiple axes with ClearType®
 [code: ranges]
</div></details><details><summary>ℹ <b>INFO:</b> Check for font-v versioning. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fontv">com.google.fonts/check/fontv</a>)</summary><div>


* ℹ **INFO** Version string is: "Version 1.010;gftools[0.9.23]"
The version string must ideally include a git commit hash and either a "dev" or a "release" suffix such as in the example below:
"Version 1.3; git-0d08353-release" [code: bad-format]
</div></details><details><summary>ℹ <b>INFO:</b> Font contains all required tables? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/required_tables">com.google.fonts/check/required_tables</a>)</summary><div>


* ℹ **INFO** This font contains the following optional tables:

	- loca

	- prep

	- GPOS

	- GSUB 

	- And gasp [code: optional-tables]
</div></details><details><summary>ℹ <b>INFO:</b> List all superfamily filepaths (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/superfamily/list">com.google.fonts/check/superfamily/list</a>)</summary><div>


* ℹ **INFO** . [code: family-path]
</div></details><br></div></details>
### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 2 | 6 | 98 | 8 | 113 | 0 |
| 0% | 1% | 3% | 43% | 4% | 50% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **PASS**
* **DEBUG**


## Fontbakery report

Fontbakery version: 0.8.10

<details><summary><b>[1] Family checks</b></summary><div><details><summary>ℹ <b>INFO:</b> Check axis ordering on the STAT table.  (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/STAT/axis_order">com.google.fonts/check/STAT/axis_order</a>)</summary><div>


* ℹ **INFO** From a total of 1 font files, 0 of them (0.00%) lack a STAT table.

	And these are the most common STAT axis orderings:
	('slnt-wght', 1) [code: summary]
</div></details><br></div></details><details><summary><b>[15] LexendMega[slnt,wght].ttf</b></summary><div><details><summary>🔥 <b>FAIL:</b> Check variable font instances (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fvar_instances">com.google.fonts/check/fvar_instances</a>)</summary><div>


* 🔥 **FAIL** fvar instances are incorrect:
- Add missing instances

| Name | current | expected |
| :--- | :--- | :--- |
| Thin Italic | N/A | wght=100.0, slnt=-15.0 |
| Thin | wght=100.0, slnt=0.0 | wght=100.0, slnt=0.0 |
| ExtraLight Italic | N/A | wght=200.0, slnt=-15.0 |
| ExtraLight | wght=200.0, slnt=0.0 | wght=200.0, slnt=0.0 |
| Light Italic | N/A | wght=300.0, slnt=-15.0 |
| Light | wght=300.0, slnt=0.0 | wght=300.0, slnt=0.0 |
| Italic | N/A | wght=400.0, slnt=-15.0 |
| Regular | wght=400.0, slnt=0.0 | wght=400.0, slnt=0.0 |
| Medium Italic | N/A | wght=500.0, slnt=-15.0 |
| Medium | wght=500.0, slnt=0.0 | wght=500.0, slnt=0.0 |
| SemiBold Italic | N/A | wght=600.0, slnt=-15.0 |
| SemiBold | wght=600.0, slnt=0.0 | wght=600.0, slnt=0.0 |
| Bold Italic | N/A | wght=700.0, slnt=-15.0 |
| Bold | wght=700.0, slnt=0.0 | wght=700.0, slnt=0.0 |
| ExtraBold Italic | N/A | wght=800.0, slnt=-15.0 |
| ExtraBold | wght=800.0, slnt=0.0 | wght=800.0, slnt=0.0 |
| Black Italic | N/A | wght=900.0, slnt=-15.0 |
| Black | wght=900.0, slnt=0.0 | wght=900.0, slnt=0.0 | [code: bad-fvar-instances]
</div></details><details><summary>⚠ <b>WARN:</b> Checking OS/2 achVendID. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/vendor_id">com.google.fonts/check/vendor_id</a>)</summary><div>


* ⚠ **WARN** OS/2 VendorID value 'NONE' is not yet recognized. If you registered it recently, then it's safe to ignore this warning message. Otherwise, you should set it to your own unique 4 character code, and register it with Microsoft at https://www.microsoft.com/typography/links/vendorlist.aspx
 [code: unknown]
</div></details><details><summary>⚠ <b>WARN:</b> Check Google Fonts glyph coverage. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage">com.google.fonts/check/glyph_coverage</a>)</summary><div>


* ⚠ **WARN** GF_TransLatin_Arabic is almost fulfilled. Missing codepoints:

	- 0x1E34 (LATIN CAPITAL LETTER K WITH LINE BELOW)


	- 0x1E35 (LATIN SMALL LETTER K WITH LINE BELOW)


	- 0x1E96 (LATIN SMALL LETTER H WITH LINE BELOW)
 

	- And 0x02BD (MODIFIER LETTER REVERSED COMMA)
 [code: missing-codepoints]
</div></details><details><summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences">com.google.fonts/check/kerning_for_non_ligated_sequences</a>)</summary><div>


* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:

	- f + f

	- f + i 

	- And i + l [code: lacks-kern-info]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure fonts have ScriptLangTags declared on the 'meta' table. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/meta/script_lang_tags">com.google.fonts/check/meta/script_lang_tags</a>)</summary><div>


* ⚠ **WARN** This font file does not have a 'meta' table. [code: lacks-meta-table]
</div></details><details><summary>⚠ <b>WARN:</b> Check font contains no unreachable glyphs (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/unreachable_glyphs">com.google.fonts/check/unreachable_glyphs</a>)</summary><div>


* ⚠ **WARN** The following glyphs could not be reached by codepoint or substitution rules:

	- gravecombinator

	- peseta.bold

	- sterling.base

	- uni004A0301

	- uni006A0301

	- uni20A6.bold

	- uni20A9.bold 

	- And uni20B1.bold
 [code: unreachable-glyphs]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure dotted circle glyph is present and can attach marks. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/dotted_circle">com.google.fonts/check/dotted_circle</a>)</summary><div>


* ⚠ **WARN** No dotted circle glyph present [code: missing-dotted-circle]
</div></details><details><summary>⚠ <b>WARN:</b> Check if OS/2 xAvgCharWidth is correct. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/os2.html#com.google.fonts/check/xavgcharwidth">com.google.fonts/check/xavgcharwidth</a>)</summary><div>


* ⚠ **WARN** OS/2 xAvgCharWidth is 584 but it should be 664 which corresponds to the average of the widths of all glyphs in the font. [code: xAvgCharWidth-wrong]
</div></details><details><summary>ℹ <b>INFO:</b> Show hinting filesize impact. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/hinting_impact">com.google.fonts/check/hinting_impact</a>)</summary><div>


* ℹ **INFO** Hinting filesize impact:

 |               | LexendMega[slnt,wght].ttf          |
 |:------------- | ---------------:|
 | Dehinted Size | 423.6kb |
 | Hinted Size   | 423.6kb   |
 | Increase      | 24 bytes      |
 | Change        | 0.0 %  |
 [code: size-impact]
</div></details><details><summary>ℹ <b>INFO:</b> Font has old ttfautohint applied? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/old_ttfautohint">com.google.fonts/check/old_ttfautohint</a>)</summary><div>


* ℹ **INFO** Could not detect which version of ttfautohint was used in this font. It is typically specified as a comment in the font version entries of the 'name' table. Such font version strings are currently: ['Version 1.010;gftools[0.9.23]'] [code: version-not-detected]
</div></details><details><summary>ℹ <b>INFO:</b> EPAR table present in font? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/epar">com.google.fonts/check/epar</a>)</summary><div>


* ℹ **INFO** EPAR table not present in font. To learn more see https://github.com/googlefonts/fontbakery/issues/818 [code: lacks-EPAR]
</div></details><details><summary>ℹ <b>INFO:</b> Is the Grid-fitting and Scan-conversion Procedure ('gasp') table set to optimize rendering? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/gasp">com.google.fonts/check/gasp</a>)</summary><div>


* ℹ **INFO** These are the ppm ranges declared on the gasp table:

PPM <= 65535:
	flag = 0x0F
	- Use grid-fitting
	- Use grayscale rendering
	- Use gridfitting with ClearType symmetric smoothing
	- Use smoothing along multiple axes with ClearType®
 [code: ranges]
</div></details><details><summary>ℹ <b>INFO:</b> Check for font-v versioning. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fontv">com.google.fonts/check/fontv</a>)</summary><div>


* ℹ **INFO** Version string is: "Version 1.010;gftools[0.9.23]"
The version string must ideally include a git commit hash and either a "dev" or a "release" suffix such as in the example below:
"Version 1.3; git-0d08353-release" [code: bad-format]
</div></details><details><summary>ℹ <b>INFO:</b> Font contains all required tables? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/required_tables">com.google.fonts/check/required_tables</a>)</summary><div>


* ℹ **INFO** This font contains the following optional tables:

	- loca

	- prep

	- GPOS

	- GSUB 

	- And gasp [code: optional-tables]
</div></details><details><summary>ℹ <b>INFO:</b> List all superfamily filepaths (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/superfamily/list">com.google.fonts/check/superfamily/list</a>)</summary><div>


* ℹ **INFO** . [code: family-path]
</div></details><br></div></details>
### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 1 | 7 | 98 | 8 | 113 | 0 |
| 0% | 0% | 3% | 43% | 4% | 50% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **PASS**
* **DEBUG**

**Comment by chrissimpkins** [comment]
Can we please document what we are waiting on to complete the onboarding of this family update?

**Comment by RosaWagner** [comment]
We are waiting corrections on the italic styles. I sent an email to the designer to know the status.

**Comment by RosaWagner** [comment]
Designer answered that he couldn't address the correction until now, will try to deliver in June

**Comment by chrissimpkins** [comment]
I spoke with Rosalie and Octavio.  We've commissioned Octavio to work on another high priority project and will push his Lexend italic revisions and out onboarding back to Q3.

**Comment by davelab6** [comment]
Per our meeting today I've moved this to Q2 so @octaviopardo has time to focus on another large project :)

**Comment by chrissimpkins** [comment]
@octaviopardo how does the timing look on this?  Something that you still plan to do in Q2?

**Comment by arjpar** [comment]
Will the Italics be true italics or obliques?

**Comment by chrissimpkins** [comment]
Moving the onboarding of these updates back to Q3.

**Comment by davelab6** [comment]
> Will the Italics be true italics or obliques?

True :)

**Comment by arjpar** [comment]
I'm looking forward to this! I can help in any way I can, if needed.

**Comment by ffrancoa** [comment]
Please implement it soon. It has been more than two years :'(

**Comment by apastuszak** [comment]
Any updates on this? Once there is an italic, I would approach Amazon to add this to the Kindle.

**Comment by vv-monsalve** [comment]
The designers are working on #3203 changes first. Once that one is solved, the Italics addition will be next.

**Comment by arjpar** [comment]
@vv-monsalve Is the work being done in the open? Is there a repository we can see to track the changes (where the work is)? I saw Octavio's repository that was updated a few weeks ago. Is it that one?

**Comment by chrissimpkins** [comment]
This issue is already tracked in https://github.com/google/fonts/issues/3203

**Comment by vv-monsalve** [comment]
@matthiasmoier, The issues for the Lexend family update were opened separately because they emerged at different points in the development. However, they are happening as a whole, so when the update tracked in the issue Chris linked above lands, it will include the italics. Keeping two issues open for one comprehensive development is no longer necessary :)

**Comment by matthiasmoier** [comment]
Perfect. Thanks for the explanation 👍

**Comment by chrissimpkins** [comment]
Sorry for the confusion Matthias!
